### ISSUE TITLE
Lost of Coins 

### ISSUE THREAD

**Issue by Paultsebee22** [ZencashOfficial/zencash-mobile#124]
I activated my wallet yesterday ,the coins sent to my account were removed on same spot and I lost coins equivalent to 279 dollars I want to know why the coins were credited and later debited on same spot without any reason or apologies thank you
Paul Tsebee

**Comment by herbertsa666** [comment]
Eu perdi 350$ triste que tenham perdido várias moedas

**Comment by angievi** [comment]
> I activated my wallet yesterday ,the coins sent to my account were removed on same spot and I lost coins equivalent to 279 dollars I want to know why the coins were credited and later debited on same spot without any reason or apologies thank you
> Paul Tsebee

Hi @Paultsebee22 can you please open a support ticket here so we can have more details: https://horizenofficial.atlassian.net/servicedesk/customer/portal/1/group/1/create/562 thanks!

**Comment by angievi** [comment]
> Eu perdi 350$ triste que tenham perdido várias moedas

Hi @herbertsa666 can you please give more details about your issue here: https://horizenofficial.atlassian.net/servicedesk/customer/portal/1/group/1/create/562

**Comment by reedniv** [comment]
Yes many people Lost of coin, maybe need update and need attention about that

**Comment by Paultsebee22** [comment]
So what am I expecting, my coins will be returned or they are lost
forever???

Which ever one its not fair to have such an experience with you guys
